features/tab-access-tabs.feature

```diff
@@ -21,6 +21,19 @@ Feature: Adjust tab stops
 
 
   @wip
+  Scenario Outline: Get tab stop values
+    Given a tab_stops having 3 tab stops
+     Then the tab stop at index <index> has position <position>
+      And the tab stop at index <index> has alignment <alignment>
+      And the tab stop at index <index> has leader <leader>
+     
+    Examples: tab stop positions
+      | index | position | alignment              | leader                  |
+      | 0     | 914400   | WD_TAB_ALIGNMENT.LEFT  | WD_TAB_LEADER.SPACES |
+      | 2     | 2743200  | WD_TAB_ALIGNMENT.RIGHT | WD_TAB_LEADER.DOTS   |
+      
+      
+  @wip
```

Comment:
These are definitely the right capabilities to be driving next, so this seems right to me.

I would break it up a bit differently though. I'll point out specifically where in individual comments. But the commits it will drive will be the same. I expect they'd be these, not necessarily in this order:

* tab: add TabStop.position getter
* tab: add TabStop.alignment getter
* tab: add TabStop.leader getter

From there you could move on to the setters, with corresponding "Set tab stop values" acceptance test(s) to drive:

* tab: add TabStop.position setter
* tab: add TabStop.alignment setter
* tab: add TabStop.leader setter

Each of which, by the way, use the getter as part of the acceptance test, hence the importance of sequencing them this way.

And then we can go back to the write-y bits of TabStops, which should go smoothly then.